infrastructure/admission/formation_generale/repository/proposition.py

```diff
@@ -160,6 +161,11 @@ def save(cls, entity: 'Proposition') -> None:
 
         candidate = Person.objects.get(global_id=entity.matricule_candidat)
 
+        financabilite_regle_etabli_par_person = None
+        if entity.financabilite_regle_etabli_par:
+            with suppress(Person.DoesNotExist):
```

Comment:
C'est possible que la personne n'existe pas ?

Reply:
Au moins si `entity.financabilite_regle_etabli_par` est None. Sinon je ne pense pas. On doit pouvoir le remplacer par un `if entity.financabilite_regle_etabli_par is not None`.